src/proficiency.h

```diff
@@ -91,6 +120,8 @@ class proficiency_set
         std::vector<proficiency_id> known_profs() const;
         std::vector<proficiency_id> learning_profs() const;
 
+        float get_proficiency_bonus( std::string category, proficiency_bonus_type proficiency_bonus ) const;
```

Comment:
Same as the change I made to the function body, don't unnecessarily copy the string argument.
```suggestion
        float get_proficiency_bonus( const std::string &category, proficiency_bonus_type proficiency_bonus ) const;
```